docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -117,31 +122,55 @@ There are several versions of CUDA on Discovery, as listed below.::
 
 Using GPUs with PyTorch
 ========================
 You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery.
 
-The following is an example of using a conda virtual environment with PyTorch for CUDA version 11.1. Make sure that you are on a GPU node before loading the environment::
+The following examples demonstrate how to build PyTorch inside a conda virtual environment for CUDA version 11.3. Make sure that you are on a GPU node before loading the environment.
 
-  module load cuda/11.1
+Lightweight installation::
+  
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  module load cuda/11.3
   module load anaconda3/2021.11
   conda create --name pytorch_env python=3.7 -y
   source activate pytorch_env
-  conda install pytorch==1.8.0 torchvision==0.9.0 torchaudio==0.8.0 cudatoolkit=11.1 -c pytorch -c conda-forge -y
+  conda install pytorch==1.9.0 torchvision==0.10.0 torchaudio==0.9.0 cudatoolkit=11.3 -c pytorch -c conda-forge -y
+  python -c'import torch; print(torch.cuda.is_available())'
+
+Heavyweight installation (with Anaconda libraries)::
+
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  module load cuda/11.3
+  module load anaconda3/2021.11
+  conda create --name pytorch_env python=3.7 anaconda -y
+  source activate pytorch_env
+  conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch -y
   python -c'import torch; print(torch.cuda.is_available())'
 
-As the latest version of Pytorch often depends the newst CUDA avaialble, please refer to the Pytorch documentation page for the installation instructions: https://pytorch.org/. 
+As the latest version of PyTorch often depends the newst CUDA avaialble, please refer to the PyTorch documentation page for the installation instructions: https://pytorch.org/. 
+
+Alternatively, you can also use our existing Pytorch build (`pytorch_env_training` environment, PyTorch version 1.8.0, works with cuda/11.1). To use it, type ::
+
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  module load anaconda3/2021.05 
+  module load cuda/11.1 
+  source activate pytorch_env_training
 
 Using TensorFlow
 ================
-We recommend that you use CUDA 10.2 with the latest version of TensorFlow.
+We recommend that you use CUDA 10.2 with the latest version of TensorFlow (TF).
 You can find the compatibility of CUDA and TensorFlow versions at the following website https://www.tensorflow.org/install/source#gpu.::
 
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
   module load anaconda3/3.7
   module load cuda/10.2
   conda create --name TF_env python=3.7 anaconda #where TF_env is the name of the conda environment
```

Comment:
source activate TF_env #load the virtual conda environment "TF_env"
conda install -c anaconda tensorflow-gpu -y #install GPU-enabled TF inside the virtual environment
python -c 'import tensorflow as tf; print(tf.test.is_built_with_cuda())' #test if GPU device is detected with TF